dag-store.js

```diff
@@ -4,13 +4,10 @@ import uuid from 'node-uuid';
 let nodes = (state = [], action = {}) => {
   switch(action.type) {
```

Comment:
So I did protect the important elements in the first part of this by making sure the label and type is up front in the assign so even if somebody tried to add a property that is required it wont use it.  I do have a use case where I want to put custom properties on the nodes.  I am experimenting uses this to build out a UI on top of workflow/orchestration engines ( Chronos via Mesos ) to allow people to build workflow type stuff with a UI.  With that I need to be able to attach custom properties to tell what each step needs to do and its nice if I can add it to the node.